charts/nri-bundle/README.md

```diff
@@ -1,11 +1,32 @@
 # nri-bundle
 
-![Version: 4.5.0](https://img.shields.io/badge/Version-4.5.0-informational?style=flat-square)
+![Version: 4.5.6](https://img.shields.io/badge/Version-4.5.6-informational?style=flat-square)
 
-A chart which groups together the individual charts for the New Relic Kubernetes solution for more comfortable deployment.
+Groups together the individual charts for the New Relic Kubernetes solution for a more comfortable deployment.
 
 **Homepage:** <https://github.com/newrelic/helm-charts>
 
+## Bundled charts
+
+This chart do not deploy anything by itself but has many charts as dependencies install and upgrade the New Relic Kubernetes
+Integration using only one chart.
+
+In case you need more information of each component this chart installs or you are and advanced user that want to install each
```

Comment:
"In case you need more information [about] each component this chart installs[,] or you are [an] advanced user that want to install each
component [separately], here is a list of components that this chart installs and where you can [find] more information about them:"

Reply:
Fixed in [#664266f](https://github.com/newrelic/helm-charts/pull/846/commits/664266f8b8372620475b4d84a853e68fe5f9ca13)